rpc/types/src/free.rs

```diff
@@ -0,0 +1,18 @@
+//! Free functions.
+
+//---------------------------------------------------------------------------------------------------- Serde
+// These are functions used for conditionally (de)serialization.
```

Comment:
Same as `defaults.rs`, this allows for conditional (de)serialization in some cases.

Examples are in the other PRs.